A3A/addons/core/functions/AI/fn_vehicleMarkers.sqf

```diff
@@ -43,8 +43,14 @@ switch (true) do {
 		_formatX = "n";
 		_color = colorTeamPlayer;
 	};
-	case (_side in [Occupants, civilian]): {
+	case (_side == civilian): {
+		_enemyX = false;
+		_formatX = "b";
```

Comment:
What does this do? Should probably make it unique just incase

Reply:
I do not know. Though I tested convoy mission and new civ event both work fine

Reply:
If you change _formatX to something like "c" it throughs error

Reply:
Hmm, should be fine as is then